plugins/device/init.go

```diff
@@ -27,6 +27,13 @@ func (p *Plugin) RestRoutes() route.Routes {
 			RequestType:  utils.GetTypeString((*deviceapi.AddDeviceReq)(nil)),
 			ResponseType: utils.GetTypeString((*deviceapi.AddDeviceResp)(nil)),
 			HandlerFunc:  deviceAddHandler},
+		route.Route{
+			Name:         "DeviceList",
+			Method:       "GET",
+			Pattern:      "/peers/{peerid}/devices",
```

Comment:
can this be changed to `/devices/{peerid}` and `/devices`, when peerid not specified then return devices from all peers else only from that peer.

For uniformity, we can change `POST /peers/{peerid}/devices` to `POST /devices/{peerid}`

Reply:
Will address listing of all devices in another PR as it requires some discussion